perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -75,9 +88,52 @@
 
 flags.DEFINE_integer('cassandra_stress_retries', 1000,
                      'Number of retries when error encountered during stress.')
+# Use ./cassandra-stress -help pop to get more details.
+# [dist=DIST(?)]: Seeds are selected from this distribution
+#  EXP(min..max):
+#      An exponential distribution over the range [min..max]
+#  EXTREME(min..max,shape):
+#      An extreme value (Weibull) distribution over the range [min..max]
+#  QEXTREME(min..max,shape,quantas):
+#      An extreme value, split into quantas, within which the chance of
+#      selection is uniform
+#  GAUSSIAN(min..max,stdvrng):
+#      A gaussian/normal distribution, where mean=(min+max)/2, and stdev
+#      is (mean-min)/stdvrng
+#  GAUSSIAN(min..max,mean,stdev):
+#      A gaussian/normal distribution, with explicitly defined mean and stdev
+#  UNIFORM(min..max):
+#      A uniform distribution over the range [min, max]
+#  FIXED(val):
+#      A fixed distribution, always returning the same value
+#  Preceding the name with ~ will invert the distribution,
+#  e.g. ~EXP(1..10) will yield 10 most.
+flags.DEFINE_enum('cassandra_stress_pop_distribution', None,
+                  ['EXP', 'EXTREME', 'QEXTREME', 'GAUSSIAN', 'UNIFORM', 'FIXED',
+                   '~EXP', '~EXTREME', '~QEXTREME', '~GAUSSIAN', '~UNIFORM'],
+                  'The population distribution cassandra-stress uses. '
+                  'By default, use no distribution.')
+
+flags.DEFINE_integer('cassandra_stress_pop_size', None,
+                     'The range of the distribution across all clients. '
+                     'By default, the size of the pop is the same as '
+                     '--num_keys.')
+
+flags.DEFINE_list('cassandra_stress_pop_parameters', [],
+                  'Additional parameters to use with. '
```

Comment:
Sorry about the incomplete sentence.

For each distribution, the corresponding command line of cassandra-stress looks like:
dist=dist_name\(start..end, additional_params\).
dist_name comes from cassandra_stress_pop_distribution;
start, end will be calculated with cassandra_stress_pop_size;
additional parameters comes from cassandra_stress_pop_parameters.

For example UNIFORM distribution only needs [min, max], which no additional parameters will be used; GAUSSIAN distribution needs either a stdvrng or mean,stdev. These will be needed to specify using the additional parameter flags.